modules/server/src/config/index.js

```diff
@@ -1,25 +1,25 @@
-import * as CONFIG from './constants';
 import { ConfigProperties } from './types';
 import getConfigFromFiles from './utils/getConfigFromFiles';
 
+export * as CONFIG from './constants';
 export { initializeSets } from './utils';
-export { CONFIG };
 
-export default async () => {
+export default async (configsSource = '') => {
+  // TODO: allow passing configs as an object, and not just a path
```

Comment:
The path to config files can now be given from the server instantiation, and not just through EnvVars.
We may also allow passing the config objects directly, so Arranger can access files hosted elsewhere?